website/src/docs/robodog.md

```diff
@@ -18,7 +18,7 @@ Robodog is an Uppy-based library that helps you talk to the Transloadit API. It
 Robodog can be downloaded from npm:
 
 ```shell
-npm install --save @uppy/robodog
+npm install  @uppy/robodog
```

Comment:
It seems this leaves two spaces here, perhaps we should replace for ` --save` instead?

Reply:
If you look closely it's not the only extra space left:

![image](https://user-images.githubusercontent.com/26752/92128405-a68fab00-ee02-11ea-8046-19845d643ed8.png)

I know i'm nitpicking but, but let's do this proper :)

Reply:
Okay I fully removed the double spaces